guides/v2.1/extension-dev-guide/versioning/codebase-changes.md

```diff
@@ -108,14 +109,19 @@ Use this table to understand what changes Magento can make and which version num
 | | Changed event property | MAJOR|
 | | Removed event property | MAJOR|
 | | Changed event ordering | MAJOR|
-| **Virtual Type**| Virtual type removed | MAJOR|
+| **Virtual Type (marked with `@api`)**| Virtual type removed | MAJOR|
 | | Virtual type added | MINOR|
-| **URL Paths** | Path removed | MAJOR|
-| | Removed/renamed a request parameter| MAJOR|
```

Comment:
@paliarush : why "removed" is MAJOR - it will not fail requests.
@kokoc : removal removes some functionality, so it's a MAJOR change.
@buskamuza : functional changes should be considered separately from technical changes.